bakerydemo/templates/base/home_page.html

```diff
@@ -26,38 +26,29 @@ <h1>{{ page.title }}</h1>
 
 <div class="container">
     <div class="row promo-row">
-        <div class="col-sm-5 promo">
-            {% if page.promo_image or page.promo_title or page.promo_text %}
-                <figure>{% image page.promo_image fill-200x200-c100 %}</figure>
-                <div class="promo-text">
-                    <h3>{{ page.promo_title }}</h3>
-                    {{ page.promo_text|safe }}
-                </div>
+        <div class="featured-cards col-sm-5 col-sm-offset-1 ">
+            {% if page.featured_section_1 %}
+                <h2 class="featured-cards__title">{{ page.featured_section_1_title }}</h2>
```

Comment:
question: does this need to be wrapped in a conditional or is it a required field?

Reply:
Good question - this is legacy code that I've left over from how the site behaved before, I will check wagtail admin to see if it's required or not

Reply:
This featured section is optional in wagtail admin, but I'll reorganise the conditional statement there as it's adding an unused `div`